interoptest/src/testcoordinator/receiver/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 Google LLC
```

Comment:
Wrong license headers, please use OpenCensus Authors.

Reply:
Good catch - not sure where I copied these headers. Fixed.